pyconpune/tasks.py

```
@@ -0,0 +1,21 @@
+from celery import Celery
+
+import logging
+
+import settings
+
+_log = logging.getLogger(__name__)
+
+
+if settings.CELERY_BROKER_URL:
+    broker_url = settings.CELERY_BROKER_URL
+else:
+    broker_url = 'amqp://localhost:5672'
```

We should not hard-code URL, raise an error if the BROKER URL is not set

---

That was for development, if anyone is feeling lazy